addons/website_sale/views/snippets.xml

```diff
@@ -97,9 +97,26 @@
             </div>
         </div>
 
-        <div data-js="ProductsSearchBar"
-            data-selector=".s_wsale_products_searchbar_input">
-            <we-button data-no-preview="true" data-open-searchbar-settings="">Products Search Bar</we-button>
+        <div data-selector=".s_wsale_products_searchbar_input">
+            <we-select string="Search filter to use by default" id="order_by" data-attribute-name="value" data-apply-to=".o_wsale_search_order_by">
+                <we-button data-select-data-attribute="">Default order</we-button>
+                <we-button data-select-data-attribute="name asc">Order by name (A-Z)</we-button>
+                <we-button data-select-data-attribute="name desc">Order by name (Z-A)</we-button>
+                <we-button data-select-data-attribute="list_price asc">Order by price: low to high</we-button>
+                <we-button data-select-data-attribute="list_price desc">Order by price: high to low</we-button>
+            </we-select>
+            <we-checkbox string="Add product suggestions" data-select-class="use_autocomplete" data-toggle="collapse" data-name="product_autocomplete_opt"/>
```

Comment:
The branch does not work on my side, please deeply test it and review before sending to review :wink: I don't see a use_autocomplete class used in the snippet. So when dropping the snippet, that option is unchecked while the feature is in fact enabled. If you have questions about the system, don't hesitate to ask :+1:

Reply:
The branch seems to work on the runbot, what part is not working?

Reply:
Like I said, I tested a bit and:
- Why toggling an unused "use_autocomplete" class on the snippet?
- Drop the snippet, the "Add product suggestion" option is unchecked, save, see that the option was in fact enabled (you type something in the input and you get product suggestion).

You don't need a select-class unless I am missing something, and you need to implement `_computeWidgetState` to set the correct state on the option.

Reply:
I see thanks, I was misusing the select-class as a way to toggle the checkbox, I just pushed the changes.